examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -664,29 +664,6 @@ THREE.LegacyGLTFLoader = ( function () {
 
 	}
 
-	function convertUint8ArrayToString( array ) {
-
-		if ( window.TextDecoder !== undefined ) {
-
-			return new TextDecoder().decode( array );
-
-		}
-
-		// Avoid the String.fromCharCode.apply(null, array) shortcut, which
-		// throws a "maximum call stack size exceeded" error for large arrays.
-
-		var s = '';
-
-		for ( var i = 0, il = array.length; i < il; i ++ ) {
-
-			s += String.fromCharCode( array[ i ] );
-
-		}
-
-		return s;
-
-	}
-
 	// Three.js seems too dependent on attribute names so globally
```

Comment:
Do you folks think we should keep updating `LegacyGLTFLoader`?

Reply:
see https://github.com/mrdoob/three.js/issues/12367#issuecomment-338784517

I would say yes, at least for the moment.